### PR TITLE
Run SLO-reporter prod from stage

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Fixes: https://github.com/thoth-station/thoth-application/issues/930

Let's run SLO-reporter prod from stage due to the issues in sending emails.

overlay -> `cnv-prod-ocp4-stage`
cronjob -> `slo-reporter-stage-prod`
CM -> `slo-reporter-stage-prod`
secret -> `slo-reporter-stage-prod`

running in `thoth-infra-stage`

sending email from internal smtp server, same as stage slo-reporter

### CODE REVIEW

File: slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sender-address: "aicoe-thoth@redhat.com"
+  email-recipients: "aicoe-thoth@redhat.com"
+  deployment-name: cnv-prod
+  amun-inspection-namespace: thoth-amun-inspection-prod
+  amun-namespace: thoth-amun-api-prod
+  backend-namespace: thoth-backend-prod
+  middletier-namespace: thoth-middletier-prod
+  frontend-namespace: thoth-frontend-prod
+  infra-namespace: thoth-infra-prod
+  storage-bucket-name: cnv-prod
```
Comment (harshad16) [inline slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:18]:
```suggestion
  storage-bucket-name: zero-prod
```

File: slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: slo-reporter-stage-prod
+data:
+  thanos-endpoint: "http://prometheus-portal-thoth-infra-prod.apps.zero.massopen.cloud"   # TODO: Use Thanos endpoint when available and set correct secret
+  smtp-server: "smtp.corp.redhat.com"
+  sender-address: "aicoe-thoth@redhat.com"
+  email-recipients: "aicoe-thoth@redhat.com"
+  deployment-name: cnv-prod
```
Comment (harshad16) [inline slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:11]:
```suggestion
  deployment-name: zero-prod
```

**Review by harshad16 — COMMENTED** [review]
/lgtm

some minor changes

File: slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: slo-reporter-stage-prod
```
Comment (harshad16) [inline slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:5]: Update this in kustomize file as well.

Comment (pacospace) [reply slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml:5]: Moved the cronjob.yaml to resources in kustomize, what I need to update for configmap?

File: slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml (hunk 2 above)
Comment (pacospace) [reply slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml]: updated! thanks!

**Pull request merged**